src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { MovieInfo } from '../../types/MovieInfo';
+
+type Props = {
+  movies: MovieInfo[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card">
```

Comment:
on the MovieCard component, you already add div with the class card. So on results, you have:
```
<div className="card">
  <div className="card">
    ...
  </div>
</div>
```